contracts/tokenomics/guages/Minter.vy

```diff
@@ -78,7 +77,7 @@ def __init__(_token: address, _controller: address, _emergency_return: address,
     self.emergency_return = _emergency_return
     self.admin = _admin
 
-    self.start_epoch_time = block.timestamp + INFLATION_DELAY - RATE_REDUCTION_TIME
+    self.start_epoch_time = block.timestamp - RATE_REDUCTION_TIME
```

Comment:
This change was made to ensure Minter can be enabled as soon as its deployed. 

The time of deployment will be used as the reward period epoch time. The rate can be changed every 2 weeks.